java-squid/src/main/java/org/sonar/java/ast/visitors/AccessorVisitor.java

```diff
@@ -35,58 +35,52 @@
 import org.sonar.plugins.java.api.tree.Tree;
 import org.sonar.plugins.java.api.tree.VariableTree;
 
-import java.util.Collections;
 import java.util.List;
 
-public class AccessorVisitor extends SubscriptionVisitor {
+public class AccessorVisitor {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)